<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="PW-HAN-063" name="Case01_12_WidgetBrick_1v6_Test_EmptyTextbox" tests="2" failures="1" timestamp="23 Sep 2019 09:44:47 GMT" time="67.005" errors="0">
  <testcase name="case07_12_emptyTextboxSubmit" time="25.322" classname="Case01_12_WidgetBrick_1v6_Test_EmptyTextbox"/>
  <testcase name="case01_06_emptyTextboxClick" time="41.683" classname="Case01_12_WidgetBrick_1v6_Test_EmptyTextbox">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	[CVV/CVC/CID 000][ErrorTextValue] expected [Please enter CVV code] but found []">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
[CVV/CVC/CID 000][ErrorTextValue] expected [Please enter CVV code] but found []
at automation.project3ds.Assertion.end(Assertion.java:26)
at Case01_12_WidgetBrick_1v6_Test_EmptyTextbox.case01_06_emptyTextboxClick(Case01_12_WidgetBrick_1v6_Test_EmptyTextbox.java:55)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- case01_06_emptyTextboxClick -->
</testsuite> <!-- Case01_12_WidgetBrick_1v6_Test_EmptyTextbox -->
